org.civicrm.reqangular/src/common/services/api/option-group.js

```diff
@@ -52,16 +52,17 @@ define([
       /**
        * Returns the values of the option groups with the given names
        *
-       * @param {string/array} groupNames
+       * @param  {String|Array} groupNames
        *   If the value is an array of names, the method will group
        *   the values by option group names
-       * @param {object} params optional parameters for the query
+       * @param  {Object} params optional parameters for the query
+       * @param  {Boolean} cache optional parameter to cache the query or not
        * @return {Promise}
        *   Resolves to an array with the values (if `groupNames` is a string)
        *   or an object with keys as the group names and values as the
        *   array of their option values (if `groupNames` is an array)
        */
-      valuesOf: function (groupNames, params) {
+      valuesOf: function (groupNames, params, cache) {
```

Comment:
remove the `params` (see above), they shouldn't have been added in the first place (i checked in our codebase and we are not using it anywhere, as we shouldn't)

Reply:
Done.